app/lib/features/comments/widgets/comments_section.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:acter/common/utils/utils.dart';
+import 'package:acter/features/comments/models.dart';
+import 'package:acter/features/comments/providers/comments.dart';
+import 'package:acter/features/comments/widgets/comments_list.dart';
+import 'package:acter/features/comments/widgets/create_comment.dart';
+import 'package:acter/features/settings/providers/settings_providers.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class CommentsSection extends ConsumerWidget {
+  final Future<CommentsManager> manager;
+  final NewCommentLocation newCommentLocation;
+  const CommentsSection({
+    super.key,
+    required this.manager,
+    this.newCommentLocation = NewCommentLocation.before,
+  });
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final provider = ref.watch(featuresProvider);
+    if (!provider.isActive(LabsFeature.comments)) {
```

Comment:
I'm still unsure which new stuff gets added in labs. What's the purpose of adding it there? Does that mean that feature isn't meant to be usable on daily basis? What's the criteria that sets lab features apart from live ones?

Reply:
> I'm still unsure which new stuff gets added in labs. What's the purpose of adding it there? Does that mean that feature isn't meant to be usable on daily basis? What's the criteria that sets lab features apart from live ones?

Fair enough. Generally my thinking is: everything that is still largely in flux and we don't want to present yet as _is_ should be a labs feature. For everything new that we want to have in `main` but not yet activated by default we should have that feature turned off by default. Once we think it is mature enough that we won't massively change it anymore, we can switch that to default-on and once that has not required any moving back, we should remove the flag entirely.

In this particular case, what I have been doing here is far from what the v1 ticket describes as a default-on-feature so I've put that behind a default-off-feature-flag. Thus we can merge it already and continue to work on it, improve the design and all and everyone, who wants to play with it can, but is clearly opting in to do so.